droid-help/src/main/resources/Web pages/FAQ.html

```diff
@@ -1,446 +1,446 @@
+            "SansSerif, Verdana, Arial, Helvetica">Frequently Asked Questions</font></b>
+          </h5>
+        </td>
+      </tr>
+    </table>
+    <table cellpadding="8" bgcolor="#eeeeee" border="0" cellspacing="0" width="100%">
+      <tr>
+        <td width="50%">
+          <a href="Welcome to DROID.html">Welcome to DROID</a> 
+        </td>
+        <td align="right" width="50%">
+          &nbsp;
+        </td>
+      </tr>
+    </table>
+    <ul>
+      <li>
+        <a href="FAQ.html#JavaVersion">What version of Java do I need?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#EarlierVersions">Can I open profiles created in earlier versions of
+        DROID?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#NotRecognising">Why isn&#39;t DROID&nbsp;recognising certain formats?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#FileTimes">Why are some file times an hour different to those on my
+        system?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#MoreAccurate">Can I configure DROID to give more accurate results?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#UseWhileProfiling">Can I continue to use information in the file system
+        while I&nbsp;am profiling it?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#HowLong">How long does profiling take?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#LeaveRunning">Can I set it up and just leave DROID running?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#PerformanceImpact">Will DROID 6 impact on the performance of my
+        systems?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#DiskSpace" id="DiskSpace" name="DiskSpace">How much disk space do I need
+        to run DROID?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#WhereLimits">Are there any limitations to where DROID can profile?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#StoreCopies">Does DROID 6 store copies of the files that it is
+        profiling?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#WhyStopped">Why has DROID stopped profiling before completion?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#CantResume">Why can&#39;t I resume a paused profile?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#NotDoneFilter" id="NotDoneFilter" name="NotDoneFilter">Why does filtering
+        on a "Not Done" status give no results?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#FilterSpeed">Why does opening some filtered folders take a long time?</a>
+      </li>
+      <li>
+        <a href="FAQ.html#StartFailure">What can I do if DROID fails to start up?</a>
+      </li>
+    </ul>
+    <h3>
+       <a id="JavaVersion" name="JavaVersion">What</a> version of Java do I need?
+    </h3>
+    <p>
+      If you are on windows, a package is provided with Java JDK. To use the version without Java bundled, you will need to have Java 8 to 12 installed on your machine in order to run DROID.
```

Comment:
We bundle the JRE from the JDK right? I think this should read as _a package is provided which also includes JRE with the droid binary_